HW_2/main.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask
+import random
+import string
+import pandas as pd
+
+app = Flask(__name__)
+
+
+@app.route("/generate_password")
```

Comment:
password should contain at least one digit, one punctuation and one upper case value. In you code sometimes password without these values.